source/crud/tests/unified/updateMany-sort.yml

```diff
@@ -0,0 +1,117 @@
+description: updateMany-sort
+schemaVersion: "1.0"
+createEntities:
+  - client:
+      id: client0
+      observeEvents:
+        - commandStartedEvent
+  - database:
+      id: database0
+      client: client0
+      databaseName: crud-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: coll0
+initialData:
+  - collectionName: coll0
+    databaseName: crud-tests
+    documents:
+      - _id: 1
+        x: 11
+      - _id: 2
+        x: 22
+      - _id: 3
+        x: 33
+tests:
+  - description: UpdateMany with sort option unsupported
+    runOnRequirements:
+      - minServerVersion: "8.0"
+    operations:
+      - name: updateMany
+        object: collection0
+        arguments:
+          filter:
+            _id:
+              $gt: 1
+          sort:
+            _id: -1
+          update:
+            $inc:
+              x: 1
+        expectError:
+          isError: true
+    expectEvents:
+      - client: client0
+        events:
+          - commandStartedEvent:
+              command:
+                update: coll0
+                updates:
+                  - q:
+                      _id:
+                        $gt: 1
+                    u:
+                      $inc:
+                        x: 1
+                    sort:
+                      _id: -1
+                    multi: true
+                    upsert:
+                      $$unsetOrMatches: false
+    outcome:
+      - collectionName: coll0
+        databaseName: crud-tests
+        documents:
+          - _id: 1
+            x: 11
+          - _id: 2
+            x: 22
+          - _id: 3
+            x: 33
+  - description: updateMany with sort option unsupported (server-side error)
```

Comment:
Client-side validation to not allow `sort` with `updateMany` suggests that we'd never communicate with the server and thus could never test for a server-side error using the CRUD API.

I suppose you could test for a server-side error by manually executing the `update` command with `runCommand`, but that'd be more of a server test and not relevant to the CRUD spec.